apps/platforms/win/nitro_reader/nitro_reader_5.py

```diff
@@ -0,0 +1,40 @@
+from talon import Module, Context, actions
+
+# --- App definition ---
+mod = Module()
+mod.apps.nitro_reader_five = """
+os: windows
+and app.name: Nitro Reader 5
+os: windows
+and app.exe: NitroPDFReader.exe
+"""
+
+# Context matching
+ctx = Context()
+ctx.matches = """
+app: nitro_reader_five
+"""
+
+
+# --- Implement actions ---
+@ctx.action_class("app")
+class app_actions:
+    # app.tabs
+    def tab_open(): actions.key("ctrl-shift-o")
+
+
+@ctx.action_class("user")
+class UserActions:
+    # user.pages
+    def page_next(): actions.key("right")
+    def page_previous(): actions.key("left")
+    def page_jump(number: int):
+        if number > 0:
+            actions.key("ctrl-g")
+            actions.edit.select_line()
+            actions.insert(str(number))
+            actions.key("enter alt:2")
+    def page_final(): actions.key("end")
+    # user.tabs
+    def tab_jump(number: int): pass
+    def tab_final(): pass
```

Comment:
What do these do?

Reply:
Tab jump isn't possible in nitro. This overwrites the default implementation of these actions so it doesn't do random stuff.

Reply:
I don't believe these actions have default implementations, do they?

Reply:
I added default implementations for these per-OS since they work for many apps, and the tag is required to opt-in. It seemed like a reasonable compromise.

https://github.com/knausj85/knausj_talon/blob/a05c3d2e3fb5cb8b1484a1e23c67413a681d6b17/code/platforms/win/app.py#L13

Reply:
Ok prob just a nit then, but shouldn't we remove these definitions if they don't have default definitions?  Prob better to just let them error out, no?

Reply:
Yeah, I tend to agree with that. It's probably better than no feedback.